src/main/kotlin/com/dansplugins/fiefs/command/fiefs/create/FiefsCreateCommand.kt

```diff
@@ -0,0 +1,56 @@
+package com.dansplugins.fiefs.command.fiefs.create
+
+import com.dansplugins.fiefs.Fiefs
+import org.bukkit.ChatColor
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.command.TabCompleter
+import org.bukkit.entity.Player
+
+class FiefsCreateCommand(private val plugin: Fiefs) : CommandExecutor, TabCompleter {
+    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
+        if (sender !is Player) {
+            sender.sendMessage("This command can only be run by a player.")
+            return false
+        }
+        val name = args.firstOrNull()
+        if (name == null) {
+            sender.sendMessage("${ChatColor.RED}" + "Please specify a name for the fief.")
```

Comment:
Appending two string templates is interesting. You could do `RED.toString() + "Please specify[...]"` or use a single string template like `"${RED}Please specify[...]"`

Reply:
Hmm yeah maybe appending two templates isn't the best. I've modified the chatcolor formatting across the project.